src/types/onyx/Form.ts

```diff
@@ -21,6 +21,8 @@ type DateOfBirthForm = Form & {
     dob?: string;
 };
 
+type FormValues = Record<string, string>;
```

Comment:
If we change it to `uknown` then multiple utils etc. down the line will start throwning errors that we're trying to pass unknown and not string since they're expecting a string. Are you sure this is the best fit here?

Reply:
Form values can be `string | boolean | Date`, so unknown is more accurate at this point. Try changing it to `string | boolean | Date`, maybe it'll work better